openmeetings-web/src/main/java/org/apache/openmeetings/web/common/tree/FileTreePanel.html

```diff
@@ -52,11 +52,16 @@ <h4 class="modal-title" id="om-js-trash-confirm-header"><wicket:message key="80"
 			<div wicket:id="tree"></div>
 		</div>
 		<div class="footer">
-			<span wicket:id="download" wicket:message="title:867"></span>
+			<span class="download" wicket:id="download" wicket:message="title:867"></span>
```

Comment:
IMO this block occupy too much space :(

I would like to get third opinion here @dkandrov WDYT?

Reply:
I think previously those numbers are just floating on the bottom without any explanation other than tool tip. 

I think the entire concept of "home" drive vs "public" drive, it just doesn't make much sense anymore. There are now 6 folders by default:
![image](https://user-images.githubusercontent.com/5152064/79625676-a7d77880-817e-11ea-9c2e-311fb4d54ea0.png)

=> So how would any user understand what those numbers are adding up now ?
Us/we understand it cause historically in OpenMeetings that is how we designed some of those drives and store them internally. 

But now: What folder belongs to home drive and what to public drive ?

Reply:
so it adds up the folders that have a "home" symbol ?
what about the group one is that part of "public" or separated ?

Reply:
There are tooltips available on mouse over :)

> But now: What folder belongs to home drive and what to public drive ?

Let's check the code :))
https://github.com/apache/openmeetings/blob/master/openmeetings-db/src/main/java/org/apache/openmeetings/db/dao/record/RecordingDao.java#L137

need to perform deeper investigation
Will do later ... :)